source/expressions.tex

```diff
@@ -2770,7 +2770,7 @@
 \indextext{\idxcode{type_info}}%
 \indexlibrary{\idxcode{type_info}}%
 \tcode{const} \tcode{std::type_info}\iref{type.info} and dynamic type \tcode{const}
-\tcode{std::type_info} or \tcode{const} \term{name} where \term{name} is an
+\tcode{std::type_info} or \tcode{const} \placeholder{name} where \placeholder{name} is an
```

Comment:
In general, I think we should use "\placeholder" for math-style single-letter variables, so these should probably we switched to "N" or "C". (The latter preferred; "N" is a namespace.)